### PR TITLE
fix storageclassname and redundancy policy

### LINKED ISSUE #170: Cluster Logging configuration issue with PVC and redundancy
1. For the PVC, if no storage class is set in the clusterlogging CR then by default it is set to an `emptyDir`, we need to set it manually [here](https://github.com/operate-first/apps/blob/master/cluster-scope/base/clusterloggings/clusterLogging.yaml#L21).
2. We also need to change the redundancy policy to `ZeroRedundancy` [here](https://github.com/operate-first/apps/blob/master/cluster-scope/base/clusterloggings/clusterLogging.yaml#L23), with `SingleRedundancy` the kibana server isn't able to communicate with the Elasticsearch instance.

cc @eranra

### PR DESCRIPTION
**Pull request by 4n4nd**
Closes #170 
Related: https://github.com/openshift/cluster-logging-operator/issues/869

### CODE REVIEW

File: cluster-scope/base/clusterloggings/clusterLogging.yaml [hunk 1]
```diff
@@ -17,10 +17,12 @@ spec:
     elasticsearch:
       nodeCount: 3
```
Comment (eranra) [inline cluster-scope/base/clusterloggings/clusterLogging.yaml:18]: change to  nodeCount: **1**

Tere is no need for 3 ES nodes at this point. This is aligned with "ZeroRedundancy"

Comment (eranra) [reply cluster-scope/base/clusterloggings/clusterLogging.yaml:18]: @4n4nd thanks for the support from my PoV if this can merge post this small fix 👍

**Review by tumido — APPROVED** [review]
Except the node count, /lgtm

File: cluster-scope/base/clusterloggings/clusterLogging.yaml (hunk 1 above)
Comment (4n4nd) [reply cluster-scope/base/clusterloggings/clusterLogging.yaml]: updated

**Comment by 4n4nd** [comment]
@tumido should be good to merge now

**Pull request merged**